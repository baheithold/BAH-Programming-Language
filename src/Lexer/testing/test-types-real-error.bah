# This file contains an error


3.14159
.5
5.
123.456
123.456.
3.14159
0.0
1.0
