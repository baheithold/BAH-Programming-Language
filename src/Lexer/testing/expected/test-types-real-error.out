REAL_TYPE 3.141590
REAL_TYPE 0.500000
REAL_TYPE 5.000000
REAL_TYPE 123.456000
ERROR, line 8: BAD_NUMBER
